src/pages/deals/Buy.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+const Buy = () => {
+  const [goodsInfo, setGoodsInfo] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/goodsData.json')
+      .then(res => res.json())
+      .then(res => {
+        setGoodsInfo(res);
+      });
+  }, []);
+
+  return (
+    <DealsBackground>
+      <ContentsWrapper>
+        {goodsInfo.map(goodsData => (
+          <GoodsInfoWrapper key={goodsData.id}>
+            <GoodsImageWrapper>
+              <GoodsImage src={goodsData.img_url} alt={goodsData.name_eng} />
+            </GoodsImageWrapper>
+            <GoodsTextWrapper>
+              <GoodsNameEng>{goodsData.name_eng}</GoodsNameEng>
+              <GoodsNameKor>{goodsData.name_kor}</GoodsNameKor>
+            </GoodsTextWrapper>
+          </GoodsInfoWrapper>
+        ))}
+      </ContentsWrapper>
+    </DealsBackground>
+  );
+};
+
+const DealsBackground = styled.div`
+  background-color: ${props => props.theme.colors.lightGray};
```

Comment:
구조분해할당 해서 사용할 수도 있습니다!
```suggestion
  background-color: ${({ theme }) => theme.colors.lightGray};
```

Reply:
감사합니다. 추후 리팩토링시 반영하도록 하겠습니다!